### PR TITLE
Aooc changes/fixes

### PR DESCRIPTION
**Pull request by TGameCo**
Adds the icon prefixes to aooc, and was supposed to make it visible to only antags. The only thing is, there is no real way to do so, as nobody put a central antagonist subset for the mind datum, nor gave a general antag setup, nor gave anything that I could do without cannibalizing most of the game's code. Ah well. Also did some color alterations and added a few small comments

### CODE REVIEW

File: code/modules/urist/verbs/antagooc.dm [hunk 1]
```diff
@@ -1,10 +1,11 @@
-var/global/normal_aooc_colour = "#FF0000"
+var/global/normal_aooc_color = "#FF3333" //Screw british speling of color. COLOR is correct. Like math. MATH is correct. MATHS is stoopid
 
 /client/verb/aooc(msg as text)
 	set name = "AOOC"
 	set category = "OOC"
+	//set hidden = 1
```
Comment (scrdest) [inline code/modules/urist/verbs/antagooc.dm]: Is there any point to keeping this as a comment?

File: code/modules/urist/verbs/antagooc.dm [hunk 2]
```diff
@@ -1,10 +1,11 @@
-var/global/normal_aooc_colour = "#FF0000"
+var/global/normal_aooc_color = "#FF3333" //Screw british speling of color. COLOR is correct. Like math. MATH is correct. MATHS is stoopid
```
Comment (scrdest) [inline code/modules/urist/verbs/antagooc.dm]: Probably better handled by a #DEFINE instead.

**Comment by scrdest** [comment]
Also, BS12 is adding something also called AOOC, but one that works differently (admin only - I don't even), so you might want to look into it so there's no confuzzlement.

File: code/modules/urist/verbs/antagooc.dm (hunk 1 above)
Comment (TGameCo) [inline code/modules/urist/verbs/antagooc.dm]: I may be able to find something later, so I kept it. For the moment though, I am stumped

**Comment by TGameCo** [comment]
Why don't they just use msay or asay? Bay is wierd

File: code/modules/urist/verbs/antagooc.dm (hunk 1 above)
Comment (scrdest) [inline code/modules/urist/verbs/antagooc.dm]: I can try to help if you tell me what you're looking for.

**Comment by scrdest** [comment]
Because both are limited to one person at a time, apparently, so it's intended as a mass-message to antags. Some people on their forums even thought it would work like our AOOC.

**Comment by TGameCo** [comment]
Hah, wierd. Where are they talking about it? I'd like to see what they're trying to do

**Comment by scrdest** [comment]
For some reason, I cannot find the commit; it's already included in the latest dev freeze.

**Comment by TGameCo** [comment]
[Found It](https://github.com/volas/Baystation12/commit/c7573a0d11a37dc2dfd14699a30a8bb687fe2eb8). It was so nice of them to tell us about this. I feel so happy that others see my ideas as so beneficial, they made their own version themselves. 

I might as well scrap my code now, next time we pull it'll break otherwise. ¯\\_(ツ)_/¯

**Comment by scrdest** [comment]
No, like I said, theirs is kinda pretty goddamn pointless most of the time, as our AOOC already includes admin->antags messages on top of what theirs has. Honestly, they should have pulled ours, they could benefit from it even more than we do.

**Comment by TGameCo** [comment]
So when the merge comes, we'll keep our version?

**Comment by scrdest** [comment]
Well, I'm not the one in charge of that, but I presume Glloyd wouldn't randomly ditch ours just because the name's the same.

File: code/modules/urist/verbs/antagooc.dm (hunk 1 above)
Comment (scrdest) [inline code/modules/urist/verbs/antagooc.dm]: Oh, I see what you were trying to do here. Hidden won't cut it - all it'd do is hide it from the panel. If you want to remove the verb itself from non-antags/admins, verbs -= /client/verb/aooc should work.

**Comment by Glloyd** [comment]
im deleting the bay one when we get it because it's fucking pointless

**Comment by Glloyd** [comment]
like a lot of things from bay these days

**Comment by TGameCo** [comment]
Why don't we just stop pulling directly from bay? Each merge seems to break more things than it adds, but I'd guess you'd know more about it glloyd

**Pull request merged**